packages/rest-hooks/src/resource/Resource.ts

```diff
@@ -76,3 +76,13 @@ export default abstract class Resource extends SimpleResource {
     });
   }
 }
+
+const proto = Object.prototype;
+const gpo = Object.getPrototypeOf;
+
+function isPojo(obj: unknown): obj is Record<string, any> {
+  if (obj === null || typeof obj !== 'object') {
+    return false;
+  }
+  return gpo(obj) === proto;
```

Comment:
this will break if the object is from another realm (iframe, or node’s vm, etc) but there’s no way to solve that for this kind of check.